src/lifecycle.ts

```diff
@@ -1,8 +1,9 @@
 import Vue from 'vue'
+import { RawLocation } from 'vue-router'
```

Comment:
Can we import vue-router here? If user doesn't install router it will generate a compile error.

Reply:
I'm verifying this locally.

Reply:
I'm still facing old problems when debug locally.

I used `npm run compile` to build the code, which succeeded.
But now the types av-ts re-exported from other packages (e.g. Vue) cause problems:
- If I use `npm i .` to install it directly, ts use types from av-ts project's own node_modules for av-ts, and my project's node_modules for my project, which cause problems.
- If I copy package.json & dist dir from av-ts to another clean location, then npm i, ts refused to use `vue` in my project's node_modules, still errors.

I guess the way ts look for type files is very tricky when there are local dependencies. I'm now taking look at verdaccio...

Reply:
What's the proper release process? I didn't see the type files in `dist` dir. Should I copy them manually from `src` dir?